### PR TITLE
Add functionality to force SSL for entire site.

### PR DESCRIPTION
**Pull request by jose-oliveira**
What was done:
- Create a setting to force SSL for entire site
- Hide granular settings if forcing SSL for entire site

### CODE REVIEW

**Comment by nerdstein** [comment]
@jose-oliveira which issue is this?

File: config/install/securepages.settings.yml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 securepages_enable: 0
 securepages_secure: 1
 securepages_switch: 1
+securepages_entire_site: 1
```
Comment (nerdstein) [inline config/install/securepages.settings.yml]: This should be a "0" by default

Comment (jose-oliveira) [inline config/install/securepages.settings.yml]: Issue 15

**Comment by nerdstein** [comment]
Linking issue #15

File: config/install/securepages.settings.yml (hunk 1 above)
Comment (nerdstein) [inline config/install/securepages.settings.yml]: Please change line 4 to a value of 0 for the module installation

File: src/Form/SecurePagesConfigForm.php [hunk 2]
```diff
@@ -84,27 +84,44 @@ public function buildForm(array $form, FormStateInterface $form_state) {
...
-    $form['securepages_pages'] = array(
+    $form['securepages_entire_site'] = array(
+      '#type' => 'radios',
+      '#title' => $this->t('Force SSL to the entire site'),
+      '#description' => $this->t('Select if SSL should be forced for the entire site or granular to specific contexts.'),
+      '#options' => array(1 => $this->t('Yes'), 0 => $this->t('No')),
+      '#default_value' => $config->get('securepages_entire_site'),
+    );
+
+    $form['securepages_granular_settings'] =  array(
```
Comment (nerdstein) [inline src/Form/SecurePagesConfigForm.php:95]: Nice!

**Comment by nerdstein** [comment]
Reviewed, merged

**Pull request merged**